src/main/webapp/scripts/comments.js

```diff
@@ -0,0 +1,36 @@
+const key = 'DEV_KEY_HERE'; //<- change as needed
+const ytDiscovery = 'https://www.googleapis.com/discovery/v1/apis/youtube/v3/rest';
+const languageDiscovery = 'https://language.googleapis.com/$discovery/rest?version=v1beta2';
+
+function loadClient() {
+    gapi.client.init({
+        'apiKey': key,
+        'discoveryDocs': [ytDiscovery, languageDiscovery],
+    }).then(function() {
```

Comment:
I believe there is a way to have the getting comments part separate from the client initiation. I think it'd be good to separate them because the function name loadClient doesn't make it clear that its loading the Client then also getting comments

Reply:
Tested some things, you don't need to put the ``` return gapi.client.youtube.commentThreads.list({``` in the then statement at all so you could do something like
```
loadClient() {
}
getComments(){
  loadClient()
  // do comment things
}
```